static/docs/user-guide/data-sharing/mounted-cache.md

```diff
@@ -0,0 +1,144 @@
+You can find more details about it in the page of
+[Large Dataset Optimization](https://dvc.org/doc/user-guide/large-dataset-optimization).
+
+## Mounted Cache Example
+
+In this example we will see how to share data with the help of a cache directory
+that is mounted through SSHFS. We are using a SSHFS example because it is easy
+to network-mount a directory with SSHFS. However once you understand how it
+works, it should be easy to implement it for other types of network-mounted
+storages (like NFS, Samba, etc.).
+
+> For more detailed instructions check out this
+> [interactive example](https://katacoda.com/dvc/courses/examples/mounted-cache).
+
+<p align="center">
+<img src="/static/img/user-guide/data-sharing/mounted-cache.png"/>
+</p>
+
+<details>
+
+### Prerequisites: Setup the server
+
+We have to do these configurations on the SSH server:
+
+- Create accounts for each user and add them to groups for accessing the Git
+  repository and the DVC storage.
+- Create a bare git repository (for example on `/srv/project.git/`) and an empty
+  directory for the DVC cache (for example on `/srv/project.cache/`).
+- Grant users read/write access to these directories (through the groups).
+
+</details>
+
+<details>
+
+### Setup each user
+
+When we have to access a SSH server, we definitely want to generate ssh key
+pairs and setup the SSH config so that we can access the server without a
+password.
+
+Let's assume that for each user we can use the private ssh key
+`~/.ssh/dvc-server` to access the server without a password, and we have also
+added on `~/.ssh/config` lines like these:
+
+```
+Host dvc-server
+    HostName host01
+    User user1
+    IdentityFile ~/.ssh/dvc-server
+    IdentitiesOnly yes
+```
+
+Here `dvc-server` is the name or alias that we can use for our server, `host01`
+can actually be the IP or the FQDN of the server, and `user1` is the username of
+the first user on the server.
+
+</details>
+
+### Mount the DVC cache
+
+With SSHFS (and the SSH configuration on the section above), we can mount the
+remote directory to `.dvc/cache/` of the project like this:
+
+```dvc
+$ mkdir -p ~/project/.dvc/cache
+$ sshfs \
+      dvc-server:/srv/project.cache/ \
+      ~/project/.dvc/cache/
+```
+
+### Optimize data management
+
+Since the cache directory is located on a mounted filesystem, we cannot use the
+_reflink_ optimization for data management. However we can use _symlinks_ (which
+work across the filesystems):
+
+```dvc
+$ dvc config cache.type 'reflink,symlink,hardlink,copy'
+$ dvc config cache.protected true
+```
+
+The configuration file `.dvc/config` should look like this:
+
+```ini
+[cache]
+type = "reflink,symlink,hardlink,copy"
+protected = true
+```
+
+This configuration is the same for all the users, so we can add it to Git in
+order to share it with the other users:
+
+```dvc
+$ git add .dvc/config
+$ git commit -m "Use symlinks if reflinks are not available"
+$ git push
+```
+
+### Sharing data
```

Comment:
I think the primary point of using shared cache usually is optimize resources - not sharing data - in sense to pass some data to another user

Reply:
> I think the primary point of using shared cache usually is optimize resources - not sharing data - in sense to pass some data to another user

Sorry, I don't understand what you are trying to say here. Can you please explain further what is the problem that you perceive?

Reply:
Kk. Let me step back a bit then. May be I'm missing the whole point of this PR still. In the index page for this new section, you write:

```
Like Git, DVC facilitates collaboration and data sharing on a distributed
environment. It makes it easy to consistently get all your data files and
directories to any machine, along with the source code.
```

And I understand this more or less. And the primary mechanism for this is just regular remotes (that's why I still confused considering that you have second PR that is about external data but also includes remotes. So both overlap in this sense but not quite focus on remotes management.

So, could you reiterate (and fix index.md?) to communicate what is the goal here? What part of the DVC workflow for end users do we cover? How does it relate to the second PR?